src/vs/workbench/browser/viewlet.ts

```diff
@@ -173,6 +177,16 @@ export class ViewletRegistry extends CompositeRegistry<Viewlet> {
 	}
 
 	/**
+	 * Registers multiple external viewlets.
+	 */
+	public registerExternalViewlets(descriptors: ViewletDescriptor[]): void {
```

Comment:
Removed this and made the loading of external viewlets this way:

- Load stocked ones and display icons on `ActivityBarPart`
- `ActivityBarPart` add in external viewlets' icons once all extensions are loaded